device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttDeviceMethod.java

```diff
@@ -21,7 +21,6 @@
 {
     private final String subscribeTopic;
     private final String responseTopic;
-    private final Map<String, DeviceOperations> requestMap = new HashMap<>();
```

Comment:
This map was from requestId (unique identifier of a method invocation) to an enum type of operation. Since this map lives in the context only of device methods, it only ever mapped to one enum value